pages/login.page.js

```diff
@@ -29,6 +30,20 @@ class LoginPage extends BasePage {
         })();
     }
 
+    async getErrorMessageAccount() {
+        return await this.getCreateAccountError().getText();
+    }
+
+    async waitForCreateAccountError() {
+        await (new WebView(element(createAccountErrorLocator), "Error Message Account")).waitForVisible(2000);
```

Comment:
no need for `new WebView(element(createAccountErrorLocator), "Error Message Account")` use the getter method